src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/net45/CustomWebBrowser.CustomWebBrowserEvent.cs

```diff
@@ -59,21 +59,22 @@ public void NavigateError(object pDisp, ref object url, ref object frame, ref ob
             // We also could tweak implementation of base class if we disagree by implementing this method.
             // This is COM events handler, defined in COM interface, however this model works as Events in .NET
             // Multiple handlers are possible, so empty method just called and do nothing.
+            
+            // Navigating event from public class could be called for internal object.
+            //       Current implementation of System.Windows.Forms.WebBrowser doesn't allow you to track who issues this event this control or IFrame,
+            //       internal IFrame will have different pDisp, so we need filter events from internal IFrames by analyzing this field:
+            //       
+            //       if ( this.webBrowser.ActiveXInstance != e.WebBrowserActiveXInstance )
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://iddp-sonarqube:9000/coding_rules#rule_key=csharpsquid%3AS125)